src/Components/Register-Login/Login/Login.css

```diff
@@ -0,0 +1,154 @@
+.login_page_container {
+    background-color: rgba(255, 255, 255, 1);
+    height: 100vh;
```

Comment:
that height should be removed from this. i only said add this to the main body and i mentioned it should be min-height